iis_action.ps1

```diff
@@ -0,0 +1,39 @@
+Param(
+    [parameter(Mandatory = $true)]
+    [string]$server,
+    [parameter(Mandatory = $true)]
+    [string]$app_pool_name,
+    [parameter(Mandatory = $true)]
+    [string]$user_id,
+    [parameter(Mandatory = $true)]
+    [SecureString]$password,
+    [parameter(Mandatory = $true)]
+    [string]$cert_path
+)
+
+$display_action = 'App Pool Status'
+$display_action_past_tense = "$display_action Returned"
+
+Write-Output "IIS $display_action"
+Write-Output "Server: $server - App Pool: $app_pool_name"
+
+$credential = [PSCredential]::new($user_id, $password)
+$so = New-PSSessionOption -SkipCACheck -SkipCNCheck -SkipRevocationCheck
+
+Write-Output "Importing remote server cert..."
+Import-Certificate -Filepath $cert_path -CertStoreLocation 'Cert:\LocalMachine\Root'
+
+$script = {
+    $app_pool = Get-IISAppPool -Name $Using:app_pool_name
+    return "$($app_pool.Name): $($app_pool.State)"
```

Comment:
I think a comment on the readme's output section would be even more helpful here since you're not just returning the state, you are appending something to it.  I actually wonder if just state would be better because people might be trying to use this programmatically to do other steps.